docs/feature-store/feature-sets.md

```diff
@@ -171,9 +171,24 @@ When defining a source, it maps to nuclio event triggers. <br>
 Note that you can also create a custom `source` to access various databases or data sources.
 
 ### Target stores
-By default the feature sets are stored as both parquet file for training and as a key value table (in the Iguazio MLOps platform) for online serving. <br>
-The parquet file is ideal for fetching large set of data for training while the key value is ideal for an online application since it supports low latency data retrieval based on key access. <br>
+By default, the feature sets are stored as both parquet file for training and as a key value table (in the Iguazio MLOps platform or a Redis DB) for online serving. <br>
```

Comment:
this is not exactly accurate.
by default, the feature sets are saved in parquet and the Iguazio NoSQL DB. The user has to change the default to be parquet and Redis